src/pgwire/src/message.rs

```diff
@@ -604,6 +607,7 @@ impl Severity {
             AdapterNotice::AlterIndexOwner { .. } => Severity::Warning,
             AdapterNotice::CannotRevoke { .. } => Severity::Warning,
             AdapterNotice::NonApplicablePrivilegeTypes { .. } => Severity::Notice,
+            AdapterNotice::PlanNotice(_) => Severity::Notice,
```

Comment:
The inner of `PlanNotice` should be expanded/matched; not worth going back through CI to fix because #20008 will need to do this.